components/brave_vpn/resources/panel/state/async.ts

```diff
@@ -86,7 +88,28 @@ handler.on(Actions.purchaseExpired.getType(), async (store) => {
       currentRegion,
       regions,
       connectionStatus: ConnectionState.DISCONNECTED,
-      expired: true
+      expired: true,
+      outOfCredentials: false,
+      stateDescription: ''
+    })
+  )
+})
+
+handler.on(Actions.outOfCredentials.getType(), async (store) => {
+  const [{ state }, { currentRegion }, { regions }] = await Promise.all([
+    getPanelBrowserAPI().serviceHandler.getPurchasedState(),
+    getPanelBrowserAPI().serviceHandler.getSelectedRegion(),
+    getPanelBrowserAPI().serviceHandler.getAllRegions()
```

Comment:
Fixed! It was in async because I copied session expired case. In that case, it seems impossible to pass a payload through or get current state. However, I changed to be a regular reducer method and then hid the regions from the dialog since it's not needed anyways.